hw2skeleton/io.py

```diff
@@ -58,14 +64,17 @@ def read_active_site(filepath):
 
                 # make a new residue if needed
                 if residue_number != r_num:
-                    residue = Residue(residue_type, residue_number)
+                    residue = Residue(residue_type, str(residue_number))
```

Comment:
This isn't needed any more because of my changes in 376deba666f11b429567e1f236fa72d29df186c0